From 35a53b9a37ca57b7d18e8f92429526c35be244b1 Mon Sep 17 00:00:00 2001
From: Janani Sankara Babu <jananis37@gmail.com>
Date: Thu, 21 Sep 2017 11:12:46 +0530
Subject: [PATCH] staging:rtl8188eu Fix remove semicolon in do {}while(0)
Git-commit: 35a53b9a37ca57b7d18e8f92429526c35be244b1
Patch-mainline: v4.15-rc1
References: FATE#326906

This patch removes the semicolon at the end of while statement in the
do while macro , inorder to avoid it behaving like compound statement.

Signed-off-by: Janani Sankara Babu <jananis37@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/staging/rtl8188eu/core/rtw_security.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/rtl8188eu/core/rtw_security.c b/drivers/staging/rtl8188eu/core/rtw_security.c
index b283a4903369..5b1ef229df2a 100644
--- a/drivers/staging/rtl8188eu/core/rtw_security.c
+++ b/drivers/staging/rtl8188eu/core/rtw_security.c
@@ -1690,4 +1690,4 @@ do {									\
 	d##1 = TE0(s##1) ^ TE1(s##2) ^ TE2(s##3) ^ TE3(s##0) ^ rk[4 * i + 1]; \
 	d##2 = TE0(s##2) ^ TE1(s##3) ^ TE2(s##0) ^ TE3(s##1) ^ rk[4 * i + 2]; \
 	d##3 = TE0(s##3) ^ TE1(s##0) ^ TE2(s##1) ^ TE3(s##2) ^ rk[4 * i + 3]; \
-} while (0);
+} while (0)
-- 
2.19.1

